dt-mapping/mapping-queries.php

```diff
@@ -1301,34 +1301,72 @@ public static function post_type_geojson( $post_type, $args = [], $offset = 0, $
         global $wpdb;
 
         //phpcs:disable
+        // Determine if post id filtering should take place.
+        $post_id_filter_sql = '';
+        $shared_user_join_sql = '';
+        $shared_user_condition_sql = '';
+        if ( isset( $args['list_all'], $args['list_all_by_user_id'] ) && !$args['list_all'] ) {
+            $post_id_filter_sql = $wpdb->prepare( "
+            SELECT * FROM
+                      (
+                        SELECT api_p.ID
+                        FROM $wpdb->posts api_p
+                        LEFT JOIN $wpdb->postmeta as field_type ON ( field_type.post_id = api_p.ID AND field_type.meta_key = 'type' )
+                        WHERE (field_type.meta_value = 'access' OR field_type.meta_value = 'user' OR field_type.meta_value = 'access_placeholder')
```

Comment:
This keeps groups from showing.
We don't need to filter by type for personal contacts